src/ButtonStyles.js

```diff
@@ -27,4 +27,8 @@ export default css`
   &:disabled {
     cursor: default;
   }
+
+  &[disabled] svg {
```

Comment:
Just curious, why did you use `[disabled]` instead of `:disabled`?

Reply:
I'm actually not sure if there's a difference, I saw `[disabled]` in an example @simurai gave when we were chatting about it and it worked so I didn't think much about it 🤔

Reply:
There's some context here if you're interested: https://stackoverflow.com/questions/20141450/should-i-use-css-disabled-pseudo-class-or-disabled-attribute-selector-or-is-i

I don't think we support IE 11 but I'll change it back to `:disabled` just for consistency sake